cogeo_mosaic/backends/dynamodb.py

```diff
@@ -53,46 +80,56 @@ def assets_for_point(self, lng: float, lat: float) -> List[str]:
         tile = mercantile.tile(lng, lat, self.quadkey_zoom)
         return self.get_assets(tile.x, tile.y, tile.z)
 
-    def info(self, fetch_quadkeys: bool = False):
+    def info(self, quadkeys: bool = False):
         """Mosaic info."""
-        if not fetch_quadkeys:
-            warnings.warn(
-                "Returning empty quadkey list, performing full scan operation might be slow and expensive on large database."
-                "You can retrieve the list of quadkey by setting `fetch_quadkeys=True`"
-            )
-
         return {
             "bounds": self.mosaic_def.bounds,
             "center": self.mosaic_def.center,
             "maxzoom": self.mosaic_def.maxzoom,
             "minzoom": self.mosaic_def.minzoom,
             "name": self.mosaic_def.name if self.mosaic_def.name else "mosaic",
-            "quadkeys": [] if not fetch_quadkeys else self._quadkeys,
+            "quadkeys": [] if not quadkeys else self._quadkeys,
         }
 
     @property
     def _quadkeys(self) -> List[str]:
         """Return the list of quadkey tiles."""
-        warnings.warn(
-            "Performing full scan operation might be slow and expensive on large database."
+        resp = self.table.query(
+            KeyConditionExpression=Key("mosaicId").eq(self.mosaic_name),
+            ProjectionExpression="quadkey",
         )
-        resp = self.table.scan(ProjectionExpression="quadkey")  # TODO: Add pagination
-        return [qk["quadkey"] for qk in resp["Items"] if qk["quadkey"] != "-1"]
+        return [
+            qk["quadkey"]
+            for qk in resp["Items"]
```

Comment:
I think you should choose a variable name other than `qk` here. I feel like `qk` should only represent a quadkey itself, while this is more like an `item` which contains a quadkey